### PR TITLE
Fix some stuff noticed during P3 study.

### PR DESCRIPTION
**Pull request by Hixie**

### CODE REVIEW

File: getting-started.md [hunk 1]
```diff
@@ -43,7 +43,9 @@ To run your app from the command-line:
 
 1. Open a terminal and change directories to the root of your app (the same directory that
 contains the `pubspec.yaml` file for your project).
-2. Run the following command.
+2. Run the following command. Once the `flutter` tool is running, if you change the app's
```
Comment (sethladd) [inline getting-started.md]: We might want to split this into a third step. After the app is running, we can ask the user to open any text editor, change a specific thing, then hit 'r'.

File: widgets.md [hunk 2]
```diff
@@ -14,10 +14,22 @@ extra_css:
...
 
-See also:
-
-* [The API reference for the `widgets` library](https://docs.flutter.io/flutter/widgets/widgets-library.html).
-* [The API reference for the `material` library](https://docs.flutter.io/flutter/material/material-library.html).
+Flutter uses a layered approach. At the highest level we provide
+[Material components](#material-design-widgets). These are implemented
+in the [`material`
+library](https://docs.flutter.io/flutter/material/material-library.html).
+That library is implemented in terms of the [`widgets`
```
Comment (sethladd) [inline widgets.md]: I'm generally in favor of not using 'that', and instead being specific. "The material library is implemented.."

File: widgets.md [hunk 3]
```diff
@@ -14,10 +14,22 @@ extra_css:
...
+library](https://docs.flutter.io/flutter/widgets/widgets-library.html).
+That library in turn is implemented in terms of the [`rendering`
+library](https://docs.flutter.io/flutter/rendering/rendering-library.html),
+and so forth. Even within each library, we have used a layered
```
Comment (sethladd) [inline widgets.md]: s/have used/used

Comment (sethladd) [inline widgets.md]: New paragraph at "Even within..." ?

File: getting-started.md (hunk 1 above)
Comment (Hixie) [inline getting-started.md]: Yeah, I considered that, but it ends up making the stuff about IntelliJ more awkward.

File: widgets.md (hunk 2 above)
Comment (Hixie) [inline widgets.md]: Fixed the first one, but I left the second to avoid making this hard to read.

File: widgets.md (hunk 3 above)
Comment (Hixie) [inline widgets.md]: Changed to "use".

Comment (Hixie) [inline widgets.md]: Done

**Comment by Hixie** [comment]
PTAL

File: getting-started.md (hunk 1 above)
Comment (sethladd) [inline getting-started.md]: Ah, then I'd create a new header before "Next Steps" called "Make a change and reload"

**Comment by sethladd** [comment]
Talked to @Hixie , we'll land this and then tweak in another PR

**Pull request merged**